lib/factor/factorQuadratic.js

```diff
@@ -249,6 +276,7 @@ function factorSumProductRule(node, symbol, aValue, bValue, cValue, negate) {
         const firstParen = Node.Creator.parenthesis(
           Node.Creator.operator('+', [rx, s]));
 
+        // 3B. Factor the second group
```

Comment:
I'm wondering if it'd be better to factor the first group and second group in two different substeps instead of the same one - what do you think?